### PR TITLE
parted: update to 3.5

### PR DESCRIPTION
**Pull request by tjhexf**

### CODE REVIEW

**Comment by tjhexf** [comment]
removed one of the patches. It doesn't seem to be necessary anymore? Or atleast that specific code piece that was patched does not seem to exist anywhere in the configure file anymore. 
Could be wrong, tho.

File: packages/parted/configure-pthreads-no-weak.patch [hunk 1]
```diff
@@ -1,12 +0,0 @@
-diff -uNr inetutils-1.9.4/configure inetutils-1.9.4.mod/configure
```
Comment (MrAdityaAlok) [inline packages/parted/configure-pthreads-no-weak.patch:1]: > removed one of the patches. It doesn't seem to be necessary anymore? Or atleast that specific code piece that was patched does not seem to exist anywhere in the configure file anymore. Could be wrong, tho.

Yes, it is wrong. Update with following patch:
```patch
--- ./configure.orig	2022-06-11 17:35:38.249779008 +0000
+++ ./configure	2022-06-11 17:37:22.492718680 +0000
@@ -17193,7 +17193,7 @@
           else
             if case "$gl_cv_have_weak" in *yes) true;; *) false;; esac; then
 
-printf "%s\n" "#define USE_POSIX_THREADS_WEAK 1" >>confdefs.h
+#printf "%s\n" "#define USE_POSIX_THREADS_WEAK 1" >>confdefs.h
 
               LIBTHREAD= LTLIBTHREAD=
             else
```

**Review by MrAdityaAlok — CHANGES_REQUESTED** [review]

**Comment by MrAdityaAlok** [comment]
Please, also change commit message to:
```
upgpkg(main/parted): update to 3.5
```
And rebase against master.

**Comment by tjhexf** [comment]
will do. Thanks!

**Review by MrAdityaAlok — APPROVED** [review]

**Pull request merged**